packages/invoices/src/OutboundShipment/DetailView/modals/hooks/useDraftOutboundLines.tsx

```diff
@@ -48,11 +48,19 @@ const createDraftOutboundLine = ({
   ...invoiceLine,
   id: invoiceLine?.id ?? generateUUID(),
   availableNumberOfPacks:
-    stockLine.availableNumberOfPacks + (invoiceLine?.numberOfPacks || 0),
+    (stockLine?.availableNumberOfPacks || 0) +
+    (invoiceLine?.numberOfPacks || 0),
   invoiceId,
-  stockLineId: stockLine.id,
+  stockLineId: stockLine?.id ?? '',
   location: invoiceLine?.location,
   expiryDate: invoiceLine?.expiryDate,
+  costPricePerPack: stockLine?.costPricePerPack ?? 0,
+  sellPricePerPack: stockLine?.sellPricePerPack ?? 0,
+  itemId: stockLine?.itemId ?? '',
+  packSize: stockLine?.packSize ?? 0,
+  onHold: stockLine?.onHold ?? false,
+  storeId: stockLine?.storeId ?? '',
+  totalNumberOfPacks: stockLine?.totalNumberOfPacks ?? 0,
```

Comment:
I changed this to be optionals so that I could utilise the same fn as a test util 🤔